inst/BS5/templates/content-authors.html

```diff
@@ -20,7 +20,7 @@ <h1>{{{pagetitle}}}</h1>
   </main>
 
   <aside class="col-md-3">
-    <nav id="toc">
+    <nav id="toc" aria-label="Table of contents">
```

Comment:
when I switch from R scripts to HTML files in the diff, the first equal sign I see without space before and after makes me jump!

Reply:
Hahaha. Also this should be translated too!